cmd.go

```diff
@@ -399,6 +399,51 @@ func deletecmd(opt Options) error {
 
 	resp, err := httpDelete(ApiURL()+fmt.Sprintf("/repos/%s/%s/releases/%d",
 		user, repo, id), token)
+
+	if resp != nil {
+		defer resp.Body.Close()
+	}
+	if err != nil {
+		return fmt.Errorf("release deletion unsuccesful, %v", err)
+	}
+
+	if resp.StatusCode != http.StatusNoContent {
+		return fmt.Errorf("could not delete the release corresponding to tag %s on repo %s/%s",
+			tag, user, repo)
+	}
+
+	return nil
+}
+
+func deletefilecmd(opt Options) error {
+	user, repo, token, tag, name := nvls(opt.DeleteFile.User, EnvUser),
+		nvls(opt.DeleteFile.Repo, EnvRepo),
+		nvls(opt.DeleteFile.Token, EnvToken),
+		opt.DeleteFile.Tag,
+		opt.DeleteFile.Name
+	vprintln("deleting...")
+
+	var rel *Release
+	var err error
+	rel, err = ReleaseOfTag(user, repo, tag, token)
+	if err != nil {
+		return err
+	}
+
+	assetId := 0
+	for _, asset := range rel.Assets {
+		if asset.Name == name {
+			assetId = asset.Id
+		}
+	}
+
+	if assetId == 0 {
+		return fmt.Errorf("coud not find asset named %s", name)
+	}
+
+	resp, err := httpDelete(ApiURL()+fmt.Sprintf("/repos/%s/%s/releases/assets/%d",
+		user, repo, assetId), token)
+
 	if resp != nil {
```

Comment:
Probably needs a different error message if/when the file deletion is successful (below).